api/v1/ytsaurus_webhook.go

```diff
@@ -45,36 +47,6 @@ func (r *Ytsaurus) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
-// TODO(user): EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-
-//+kubebuilder:webhook:path=/mutate-cluster-ytsaurus-tech-v1-ytsaurus,mutating=true,failurePolicy=fail,sideEffects=None,groups=cluster.ytsaurus.tech,resources=ytsaurus,verbs=create;update,versions=v1,name=mytsaurus.kb.io,admissionReviewVersions=v1
-
-var _ webhook.Defaulter = &Ytsaurus{}
-
-// Default implements webhook.Defaulter so a webhook will be registered for the type
-func (r *Ytsaurus) Default() {
-	ytsauruslog.Info("default", "name", r.Name)
-
-	// Set anti affinity for masters
-	if r.Spec.PrimaryMasters.Affinity == nil {
-		r.Spec.PrimaryMasters.Affinity = &corev1.Affinity{}
-	}
-	if r.Spec.PrimaryMasters.Affinity.PodAntiAffinity == nil {
-		r.Spec.PrimaryMasters.Affinity.PodAntiAffinity = &corev1.PodAntiAffinity{
-			RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
-				{
-					LabelSelector: &metav1.LabelSelector{
-						MatchLabels: map[string]string{
-							consts.YTComponentLabelName: fmt.Sprintf("%s-%s", r.Name, "yt-master"),
-						},
-					},
-					TopologyKey: "kubernetes.io/hostname",
-				},
-			},
-		}
-	}
-}
-
```

Comment:
Do this affinities not needed or set somewhere else?

Reply:
Not always needed. I've added validation instead.